fonts.less

```diff
@@ -49,9 +49,10 @@
 .font-face('Main', bold, italic);
 .font-face('Main', normal, italic);
 .font-face('Main', normal, normal);
-// .font-face('Math', bold, italic);
+//.font-face('Math', bold, normal);
+.font-face('Math', bold, italic);
 .font-face('Math', normal, italic);
-// .font-face('Math', normal, normal);
+.font-face('Math', normal, normal);
```

Comment:
> AFAIK, default LaTeX and KaTeX uses the same font as the text mode, Computer Modern Roman(cmr) and Computer Modern (Roman) Bold Extended(cmbx), for \mathrm and \mathbf, respectively.

So I think upright(Roman) `KaTeX-Math` and upright bold is not necessary.

Reply:
OK, I agree, and commented this back out.  We're still generating `KaTeX-Math-Regular` (and I still don't know how to generate `KaTeX-Math-Bold` if we wanted it -- would need a mapping).  Do we feel confident enough to remove this font from the repo?  Perhaps better to wait for more confirmation, and just publish this as a solution to `\mathbold`.